app/templates/views/dashboard/dashboard.html

```diff
@@ -54,75 +54,88 @@ <h2 class="line-height-sans-2 margin-bottom-0 margin-top-4">
 
     {% if current_user.has_permissions('manage_service') %}{% endif %}
 
-    {{ ajax_block(partials, updates_url, 'template-statistics') }}
-    <h2 class="margin-top-4 margin-bottom-1">Recent Batches</h2>
-    <div class="table-overflow-x-auto">
-      <table class="usa-table usa-table--borderless job-table">
-        <thead class="table-field-headings">
-          <tr>
-            <th scope="col" class="table-field-heading-first">
-              <span>File name</span>
-            </th>
-            <th scope="col" class="table-field-heading">
-              <span>Template</span>
-            </th>
-            <th scope="col" class="table-field-heading">
-              <span>Job status</span>
-            </th>
-            <th scope="col" class="table-field-heading">
-              <span>Sender</span>
-            </th>
-            <th scope="col" class="table-field-heading">
-              <span># of Recipients</span>
-            </th>
-            <th scope="col" class="table-field-heading">
-              <span>Report</span>
-            </th>
-          </tr>
-        </thead>
-        <tbody>
-          {% if jobs %}
-            {% for job in jobs[:5] %}
+    <div class="table-container">
```

Comment:
Ideally, we can refactor the tables to have only one to manage/load. We could add in conditionals, rather than creating a new table

Reply:
you can combine the two tables into one and use JavaScript to dynamically update the table content based on the selected option

Reply:
@heyitsmebev that's definitely a good idea!  Please feel free to write up an issue to capture that, this way we don't lose track of making this easier and better for ourselves. 🙂